.github/PULL_REQUEST_TEMPLATE/task.md

```diff
@@ -0,0 +1,20 @@
+
+[//]: # (Lines starting with [//]: are considered to be comments. You do not have to delete those lines, they are not rendered upon release.)
+
+**Related Task**: #
```

Comment:
Add something like "Please provide the id of the issue of the task you've solved. This should be used in order to get information about the task itself."